src/LibraryManager.Contracts/FileHelpers.cs

```diff
@@ -245,21 +254,33 @@ public static bool DeleteFiles(IEnumerable<string> filePaths)
         /// Deletes folders if empty
         /// </summary>
         /// <param name="folderPaths"></param>
+        /// <param name="rootDirectory"></param>
         /// <returns></returns>
-        public static bool DeleteEmptyFoldersFromDisk(IEnumerable<string> folderPaths)
+        public static bool DeleteEmptyFoldersFromDisk(IEnumerable<string> folderPaths, string rootDirectory)
```

Comment:
>public [](start = 8, length = 6)

This should be private since it's only used in this class

Reply:
Yes it is

---
In reply to: [201165106](https://github.com/aspnet/LibraryManager/pull/286#discussion_r201165106) [](ancestors = 201165106)